### PR TITLE
commit models and server mds

### PR DESCRIPTION
**Pull request by BlakeMCavender**
There are two markdown files here:
- server.md
- model.ms

The server.md markdown file describes the server folder within our project, and the following files within the server folder

The model.md markdown file describes the models used within our back-end, along with the properties of the models and how beta affects the models.

### CODE REVIEW

File: Documentation/models.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - beta of 0.50
+- b0.75_StegoModel
+  - beta of 0.75
+
+# Choosing the Correct Model
+A model chossen closest to the current beta value is done within the `serverUtils.py` program with the `./python` folder.
+
+**serverUtils.py:**
+
+```javascript
```
Comment (EricABradshaw) [inline Documentation/models.md:32]: python?

**Review by EricABradshaw — APPROVED** [review]
Looks good

File: Documentation/models.md (hunk 1 above)
Comment (BlakeMCavender) [reply Documentation/models.md:32]: Correct. Let me change that. Thank you.

**Review by jacobRidenour — APPROVED** [review]
looking good

**Review by EricABradshaw — APPROVED** [review]

**Pull request merged**